### PR TITLE
Axsuarez/swagger spec

### LINKED ISSUE #5945: Spec for updating swagger
Create a spec to define the process of updating the swagger file. Now that we decided to get rid of individual swagger per language repo, we need a process to keep track of the changes.

### PR DESCRIPTION
**Pull request by axelsrz**
Fixes #5945 

Initial proposal for new swagger process spec.

### CODE REVIEW

File: specs/botframework-protocol/channel-update-spec.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Updating the botframework-channel swagger file
+
+## Abstract
+This document describe specific guidelines for upding the [swagger definiton file](https://github.com/microsoft/botframework-sdk/blob/master/specs/botframework-protocol/botframework-channel.json) hosted in this repo. These guidelines specify the steps and requirements to mainting a consistent source of truth that describes the state of connector-features in each SDK codebase. This source of truth is the combination of 2 concepts:
```
Comment (EricDahlvang) [inline specs/botframework-protocol/channel-update-spec.md:4]: spelling: **mainting**

File: specs/botframework-protocol/channel-update-spec.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+### 1. Submiting the PR
+Every schema PR MUST inlude **only** the schema update. This means that the only file changed in that PR should be the swagger file.
+
+The schema PR MAY include changes in diferent parts of the json, it SHOULD NOT be one schema PR per change in json object, but every schema PR including content change (not just format, like indentation) MUST increment the 'version' object inside the 'info' base-level object. (Specific version system is still WIP)
+
+The schema PR MUST be tagged with the schema label.
+
+### 2. Tracking issues
+
+Once a schema PR is approved, one implementation issue MUST be opened for **each enpoint changed** per language repo. The implementation issue MUST reference the schema PR that it comes from, as well as include the schema label and a version label. The implementation issue MAY provide additional context in the body text.
```
Comment (EricDahlvang) [inline specs/botframework-protocol/channel-update-spec.md:54]: spelling: **enpoint**

File: specs/botframework-protocol/channel-update-spec.md (hunk 1 above)
Comment (cleemullins) [reply specs/botframework-protocol/channel-update-spec.md:4]: @EricDahlvang As a matter of "The easy way vs The Hard way" you can edit this directly yourself in the browser while doing a PR review. I've started doing that, just to fix the back-and-forth and make things go faster...

**Comment by cleemullins** [comment]
I have feedback, but think we can merge this then treat the feedback as issues. 

@johnataylor, @gabog would love your opinions on this speclet.

File: specs/botframework-protocol/channel-update-spec.md (hunk 1 above)
Comment (EricDahlvang) [reply specs/botframework-protocol/channel-update-spec.md]: @cleemullins k, that's definitely faster!

File: specs/botframework-protocol/channel-update-spec.md (hunk 2 above)
Comment (EricDahlvang) [reply specs/botframework-protocol/channel-update-spec.md]: fixed

**Pull request merged**